usermods/TTGO-T-Display/README.md

```diff
@@ -87,5 +69,7 @@ xtensa-esp32-elf-g++: fatal error: no input files
 ```
 try building again. Sometimes this happens on the first build attempt and subsequent attempts build correctly.
 
+Once the compilation is done and loaded onto the TTGO T-Display module, you may need to complete the initial setup of WLED via the AP interface, configure the WiFi settings, let it restart and then go to the LED preferences option in the WLED web interface.  The Data GPIO pin needs to be set to pin 2 (or whatever pin you used), and then reboot WLED on the device.  This should get the display working.  I am unable to get the overrides to change the initial pin setting of 16 (which conflicts with the display).
```

Comment:
"I am unable to get the overrides to change the initial pin setting of 16 (which conflicts with the display)."

@spiff72 using `-D DATA_PINS=2` did not work?
Actually any build_flags for pin assignment will be overwritten by your cfg.json, so you'll need to wipe your device first if you want to test default pins assigned in build_flags.

https://github.com/Aircoookie/WLED/blob/ae4de2782a10a1db6cb007558af9ec81f339366c/platformio_override.sample.ini#L61-L70